### ISSUE TITLE
Tooltips not announced by screen reader (#3525)

### ISSUE THREAD

**Issue by hippee-lee** [clr-team/planning#42]
## Description of work
Update tooltips with attributes that make them accessible to screen readers.

_Description of the work to be done, in a clear but succinct manner._
tbd

## Issue(s)

* https://github.com/vmware/clarity/issues/3525

## Reviewers

* Bozhidar

* Jeeyun
* Jeremy

## Definition of done

* [ ] Tests completed
* [ ] Reviewed by peers
* [ ] Design sign off (if necessary)
* [ ] Merged into master/topic branch
* [ ] Backport (if necessary)

**Comment by clane** [comment]
Also ensure that they do not disappear when you hover over them, can be dismissed without moving the pointer hover or focus (this is in case Clarity consumers use them in a way that covers up content when the tooltips are visible), and that the tooltip stays visible until the hover or focus trigger is removed. All this is documented at  https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus

**Comment by hippee-lee** [comment]
https://github.com/vmware/clarity/pull/3669
